src/Aloha/Twilio/Commands/TwilioMmsCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Aloha\Twilio\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+use Aloha\Twilio;
+
+class TwilioMmsCommand extends Command {
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'twilio:mms';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Twilio command to test Twilio API Integration.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
```

Comment:
You have this same block in the other two command files as well. My new commit has updated the description to match the call.